forge-ai/src/main/java/forge/ai/CreatureEvaluator.java

```diff
@@ -220,76 +328,68 @@ else if (c.hasKeyword(Keyword.WITHER)) {
             if (c.isTapped()) {
                 value = addValue(50 + (c.getCMC() * 5), "tapped-useless"); // reset everything - useless
             } else {
-                value -= subValue(50, "doesnt-untap");
+                value += subValue(50, "doesnt-untap");
             }
         } else {
-            value -= subValue(10 * c.getCounters(CounterEnumType.STUN), "stunned");
+            value += subValue(10 * c.getCounters(CounterEnumType.STUN), "stunned");
         }
         if (c.hasSVar("EndOfTurnLeavePlay")) {
-            value -= subValue(50, "eot-leaves");
+            value += subValue(50, "eot-leaves");
         } else if (c.hasKeyword(Keyword.CUMULATIVE_UPKEEP)) {
-            value -= subValue(30, "cupkeep");
+            value += subValue(30, "cupkeep");
         } else if (c.hasStartOfKeyword("UpkeepCost")) {
-            value -= subValue(20, "sac-unless");
+            value += subValue(20, "sac-unless");
         } else if (c.hasKeyword(Keyword.ECHO) && c.cameUnderControlSinceLastUpkeep()) {
-            value -= subValue(10, "echo-unpaid");
+            value += subValue(10, "echo-unpaid");
         }
         if (c.hasKeyword(Keyword.FADING)) {
-            value -= subValue(20 / (Math.max(1, c.getCounters(CounterEnumType.FADE))), "fading");
+            value += subValue(20 / (Math.max(1, c.getCounters(CounterEnumType.FADE))), "fading");
         }
         if (c.hasKeyword(Keyword.VANISHING)) {
-            value -= subValue(20 / (Math.max(1, c.getCounters(CounterEnumType.TIME))), "vanishing");
+            value += subValue(20 / (Math.max(1, c.getCounters(CounterEnumType.TIME))), "vanishing");
```

Comment:
I do believe this breaks and would just increase now...

Reply:
can't the `-` be moved inside the addValue call instead so the debug output in `SimulationCreatureEvaluator` will be correct?